omnij-rpc/src/integ/groovy/foundation/omni/test/rpc/reorgs/SendToOwnersReorgSpec.groovy

```diff
@@ -164,6 +165,7 @@ class SendToOwnersReorgSpec extends BaseReorgSpec {
         // secondOrphanedTx.confirmations == -1 TODO: activate after Omni Core adjustment
 
         when: "creating a third STO transaction"
+        requestBitcoin(actorAddress, Coin.CENT)  // maybe use startBTC instead
```

Comment:
@msgilligan: is there something like `Coin.of(startBTC)`, which works with `BigDecimal`?